### PR TITLE
xplat: enable -test build type

### PR DESCRIPTION
**Pull request by jianchun**
Simple changes to enable "test" build type on cross-platform.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -27,6 +27,7 @@ PRINT_USAGE() {
     echo "    --icu         : Path to ICU include folder (see example below)"
     echo "    --jobs, -j    : Multicore build (i.e. -j=3 for 3 cores)"
     echo "    --ninja, -n   : Build with ninja instead of make"
+    echo "    --test, -t    : Test build (by default Release Build)"
```
Comment (obastemur) [inline build.sh]: `--test` feels like it's going to run tests. `--test-build` ?

Comment (digitalinfinity) [inline build.sh]: agree with @obastemur. Alternatively, --fretest if you want something shorter? Although that has msft connotations with the usage of fre, so I'm fine with whatever here.

Comment (jianchun) [inline build.sh]: Updated to --test-build. Thanks!

File: build.sh [hunk 2]
```diff
@@ -27,6 +27,7 @@ PRINT_USAGE() {
     echo "    --icu         : Path to ICU include folder (see example below)"
     echo "    --jobs, -j    : Multicore build (i.e. -j=3 for 3 cores)"
     echo "    --ninja, -n   : Build with ninja instead of make"
+    echo "    --test-build, -t  : Test build (by default Release Build)"
```
Comment (dilijev) [inline build.sh]: I think you could update the whole stretch of options here so the colons are aligned.

Comment (jianchun) [inline build.sh]: > I think you could update the whole stretch of options here so the colons are aligned.

Done. Thanks!

**Comment by dilijev** [comment]
LGTM

**Comment by jianchun** [comment]
Thanks @obastemur @digitalinfinity @dilijev

**Pull request merged**